src/types/models.ts

```diff
@@ -19,65 +12,47 @@ export type IconSize = L.PointExpression;
 
 export type IconAnchor = L.PointExpression;
 
-export type Result = string | number | null | JSX.Element;
-
-export type Host = {
-  avatar: string;
-  username: string;
-  userStatus: string;
-  text: string;
-}
-
-export type Review = {
-  id: number;
-  avatar: string;
-  name: string;
-  date: string;
-  text: string;
-};
-
-export type Reviews = Review[];
-
-export type Image = {
-  id?: number;
-  image: string;
-}
-
-export type Feauture = {
-  id?: number;
-  type: string;
-  feauture: string;
-}
-
-export type Option = {
-  id?: number;
-  option: string;
-}
-
 export type Coordinates = {
   latitude: number;
   longitude: number;
+  zoom: number;
 }
 
 export type Offer = {
-  images: Image[];
-  imageSource: string;
+  id: string;
+  title: string;
+  type: string;
+  price: number;
+  city: City;
+  location: Coordinates;
+  isFavorite: boolean;
   isPremium: boolean;
-  name: string;
-  isBookmarks: boolean;
   rating: number;
-  ratingValue: number;
-  feautures: Feauture[];
-  price: number;
-  options: Option[];
-  id: string;
-  placeType: string;
-  host: Host;
-  reviews: Reviews;
-  town: string;
-  coordinates: Coordinates;
+  previewImage: string;
 };
 
+export type CurrentOffer = {
+  id: string;
+  title: string;
+  type: string;
+  price: number;
+  city: City;
+  location: Coordinates;
+  isFavorite: boolean;
+  isPremium: boolean;
+  rating: number;
+  description: string;
+  bedrooms: number;
+  goods: string[];
+  host: {
+    name: string;
+    avatarUrl: string;
+    isPro: boolean;
+  };
+  images: string[];
+  maxAdults: number;
+} | null;
```

Comment:
Кажется, это имеет мало смысла прямо в типе указывать, что он `null`. 

```suggestion
};
```